documentation/user_docs/docs/API_Reference.rst

```diff
@@ -0,0 +1,32 @@
+API Reference
+#############
+
+.. `sphinxcontrib.matlabdomain` can only handle new-style classes 
+.. (defined by `classdef`) - so cannot autodocument sqw class at present
```

Comment:
Ah... so the docs are built from the release branches (`rel_3_6` at the moment) and I don't think they have all the new classes yet... So the documented classes are just random ones I put in because I thought they might be useful(ish)... (the only one which is actually referenced in the text is `SQWDnDBase` - specifically the `disp2sqw_eval` and related methods.

Unlike with Python, the `sphinxcontrib.matlabdomain` can only document classes you specify (in Python you can just give it a module name and it will autodocument all the classes defined by that module).